src/main/Port.Adapter/UI/Views/Blazor.Common/TreeView.razor

```diff
@@ -316,6 +319,80 @@
             "";
     }
 
+    private async Task<string> Hover(TreeNeuronViewModel child, bool recursive = false, string id = "")
+    {
+        try
+        {
+            return await ProcessHoverOrUnhover(child, recursive, id, true);
+        }
+        catch (Exception ex)
+        {
+            return ex.Message;
+        }
+    }
+
+    private async Task<string> Unhover(TreeNeuronViewModel child, bool recursive = false, string id = "")
+    {
+        try
+        {
+            return await ProcessHoverOrUnhover(child, recursive, id, false);
+        }
+        catch (Exception ex)
+        {
+            return ex.Message;
+        }
+    }
+
+    private async Task<string> ProcessHoverOrUnhover(TreeNeuronViewModel child, bool recursive, string id, bool isHover)
```

Comment:
Can we rename this to ProcessHoverCore? If the function HoverCore on line 391 is no longer necessary?